.github/workflows/test-application.yaml

```diff
@@ -73,26 +83,21 @@ jobs:
                   restore-keys: |
                       ${{ runner.os }}-composer-
 
-            - name: Require phpstan dependency
-              if: ${{ matrix.phpstan }}
-              run: composer require --dev phpstan/phpstan --no-interaction --no-update
-
             - name: Install dependencies
               run: |
                   composer validate --strict
                   composer update ${{ matrix.composer-flags }}
 
-            - name: Create database
-              run: |
-                  Tests/app/console doctrine:database:create
-                  Tests/app/console doctrine:schema:update --force
+            - name: Bootstrap test environment
+              run: composer bootstrap-test-environment
               env: ${{ matrix.env }}
 
-            - name: PHPStan
-              if: ${{ matrix.phpstan }}
-              run: vendor/bin/phpstan analyse ./ --level 1 -c phpstan.neon
+            - name: Lint
+              if: ${{ matrix.lint }}
+              run: composer lint
```

Comment:
Is there a `lint-container` script in the `composer.json`

Reply:
https://github.com/sulu/SuluFormBundle/blob/2.0/composer.json#L77 yep :)

Reply:
I mean https://symfony.com/blog/new-in-symfony-4-4-service-container-linter this 🙂

Reply:
I'm sorry, forgot how we did this before.
https://github.com/sulu/SuluArticleBundle/blob/master/.github/workflows/test-application.yaml#L169-L180
This ensures that the container is being linted in all environments, we should do the same here ...

Reply:
Oh ok thank you

Reply:
wouldn't it be more convenient to make a composer.json script for this ?

Reply:
yes, you can do that if you want